guides/source/getting_started.md

```diff
@@ -148,22 +148,27 @@ To use this generator, open a terminal, navigate to a directory where you have
 rights to create files, and run:
 
 ```bash
-$ rails new blog
+$ rails new blog --skip-bundle
```

Comment:
I'm not sure how to feel about these bundler changes :/

On one hand, I agree that this may help if you have installed ruby into system paths or somewhere you may not have access to install gems. But on the other hand, I think the majority of users don't run into these types of issues (using a version manager/gems aren't in a root controlled path) and this adds additional cognitive complexity really early on in a very beginner guide.